### ISSUE TITLE
Crash when username detection is line-number

### ISSUE THREAD

**Issue by WasabiPizza** [geluk/pass-winmenu#66]
The title should say it all. I (un)commented the relevant fields but it always crashes, unfortunately I don't know how to use the regex method so I would really appreciate if you could look into it. Thanks.

**Comment by geluk** [comment]
Thank you for your report! I discovered the cause of this issue and I am working on a fix, which I will publish soon.
For now, you can work around the issue by placing your username on line number 4 or higher, and modifying the cofiguration file correspondingly by setting `line-number` to the value you've chosen.

**Comment by geluk** [comment]
I have created a new build that fixes this issue. Could you download it and check if it resolves the problem you're having?
You can download it here: https://ci.appveyor.com/api/buildjobs/l0roxjivd4jc76t0/artifacts/pass-winmenu%2Fbin%2FRelease%2Fpass-winmenu.exe

It can be used as a drop-in replacement for the `pass-winmenu.exe` you are using currently.

**Comment by WasabiPizza** [comment]
Thank you very much for the update! Unfortunately it crashes at startup, this is the log.

```
[12:44:14.643] [DBG] ------------------------------
[12:44:14.646] [DBG] Starting pass-winmenu v1.10.1-18-gc4b6366
[12:44:14.646] [DBG] ------------------------------
[12:44:14.650] [DBG] Enabled security protocols: Tls, Tls11, Tls12, Tls13
[12:44:14.905] [ERR] Could not start pass-winmenu: An exception occurred.
[12:44:14.912] ArgumentException (Nome di directory  non valido.) in :0:0
[12:44:14.912]   -> in System.IO.FileSystemWatcher..ctor -- :0:0
[12:44:14.912]   -> in PassWinmenu.Configuration.ConfigManager.EnableAutoReloading -- :0:0
[12:44:14.912]   -> in PassWinmenu.Program.LoadConfigFile -- :0:0
[12:44:14.912]   -> in PassWinmenu.Program.Initialise -- :0:0
[12:44:14.912]   -> in PassWinmenu.Program.Start -- :0:0
```

"Nome di directory  non valido" should mean something like invalid directory name.

**Comment by geluk** [comment]
Sorry for leaving you waiting for so long, I didn't get around to it right away, and forgot about it afterwards.
Nonetheless, I have fixed the second issue in a new build, which you can download here: https://ci.appveyor.com/api/buildjobs/6oqbj5sory9tlxtk/artifacts/pass-winmenu%2Fbin%2FRelease%2Fpass-winmenu.exe

Let me know if this fixes your issue; I'll create a new release soon whcih will include the fix for this and a few other issues.

**Comment by geluk** [comment]
Fixed in [v1.11](https://github.com/geluk/pass-winmenu/releases/tag/v1.11). Should you still encounter any issues, please let me know!
